src/pkg/cli/client/byoc.go

```diff
@@ -142,12 +144,64 @@ func (b *byocAws) Deploy(ctx context.Context, req *v1.DeployRequest) (*v1.Deploy
 		return nil, err
 	}
 
+	// Create the subdomain delegation and pass the subdomain to the CD task
+	if b.customDomain == "" {
+		log.Printf("no custom domain name provided, obtain and delegate a subdomain zone.\n")
```

Comment:
we have our own output funcs debug/warn/info/error in https://github.com/defang-io/defang/blob/89df1e2d19269a9d6ea4ab74b2b0895634a6f624/src/pkg/cli/colorizer.go#L103